apisix/discovery/nacos.lua

```diff
@@ -80,8 +83,16 @@ local schema = {
 }
 
 
-local _M = {}
+local _M = {
+    version = 0.2,
```

Comment:
We don't need to add the version number.

Reply:
Thx. I fixed.